packages/odyssey-react-mui/src/labs/SideNav/SideNavHeader.tsx

```diff
@@ -37,33 +43,33 @@ const SideNavHeaderContainer = styled("div", {
   display: "flex",
   justifyContent: "space-between",
   alignItems: "center",
-  paddingLeft: odysseyDesignTokens.Spacing4,
-  paddingRight: odysseyDesignTokens.Spacing4,
-  paddingTop: odysseyDesignTokens.Spacing3,
-  paddingBottom: odysseyDesignTokens.Spacing3,
+  padding: odysseyDesignTokens.Spacing4,
+  width: "100%",
+
+  h2: {
+    margin: 0,
+    width: "100%",
+  },
```

Comment:
Why are we targeting `h2` here? Do we even have `h2`s on the page? Can we target a specific element instead?

Reply:
This is how we ensure that the Skeleton loader for the sidenav heading is full-width. I've updated it to target `MuiTypography-root` instead, to be resilient against future changes to the element.